src/Tizen.NUI/src/public/DragAndDrop/DragEvent.cs

```diff
@@ -62,6 +62,24 @@ public struct DragData
         public string Data { get; set; }
     }
 
+    /// <summary>
+    /// This specifies drag info
+    /// </summary>
+    // Suppress warning : This struct will be used data of callback, so override equals and operator does not necessary.
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1815: Override equals and operator equals on value types")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct DragInfo
+    {
+        /// <summary>
+        /// The mime types of drag data
+        /// </summary>
+        public string [] MimeTypes { get; set; }
```

Comment:
이번 PR을 통해서 다양한 MimeType을 동시에 추가할 수 있게 되어 좋은것 같습니다 👍 
관련해서 한 가지 궁금한 점이 있는데 DragInfo을 별도로 추가하신 이유가 하위 호환성때문일까요?
만약 그렇다면, 사용성 측면의 관점에서 새로운 struct를 추가하기보다는 기존 DragData 안으로 해당 type을 추가하는게 좀 더 좋은 구조일 것 같습니다.
이렇게 추가 후에 기존의 MimeType가 필요하지 않는 상황이라면 deprecated 하는게 좋을 듯 합니다.
```suggestion
    public struct DragData
    {
        /// <summary>
        /// The mime type of drag data
        /// </summary>
        public string MimeType { get; set; }
        
        /// <summary>
        /// The mime types of drag data
        /// </summary>
        public string [] MimeTypes { get; set; }
```

Reply:
추가로 이러한 구조로 갔을때 data 역시 여러 값을 처리할 수 있으면 좋을것 같은데
이 부분에 대해서는 어떻게 생각하시나요?
안드로이드의 ClipData 부분을 참고하면 도움이 될 것 같습니다.
https://developer.android.com/reference/android/content/ClipData

안드로이드에서는 ClipData 안에 여러 ClipItem들과 mimeType으로 구성되어 있어서
여러 데이터를 동시에 받아서 처리할 수 있는 구조로 되어있는 것 같습니다.
관련해서 저희도 비슷한 구조로 변경되면 좋을 것 같은데 어떻게 생각하시나요?

Reply:
@taehyub 
추가로 MimeTypes와 대응되는 data를 Dictionary로 변경하면
데이터 관리 측면에서 도움이 될 것 같은데 어떻게 생각하시나요?

클린코드 관점 중 의도 명확성 및 코드 간결성 관점에서 기존에 있는 라이브러리를 활용하는게 좀 더 좋을것 같아 의견 드립니다.
Dictionary 설명과 간단한 적용 예시에 대해서 아래 링크 추가드립니다.

Dictionary 설명:
https://learn.microsoft.com/ko-kr/dotnet/api/system.collections.generic.dictionary-2?view=net-8.0

```suggestion
Dictionary<string, string> dataMap = new Dictionary<string, string>();
dataMap.Add("mimeType1","mimeData1");
dataMap.Add("mimeType2","mimeData2");
```

Reply:
@tscholb 
좋은 의견 입니다. MimeType과 대응되는 Data를 Dictionary로 묶는 다면 코드의 응집도가 높아질 것 같네요^^

Reply:
```C#
public struct DragData
{
    /// <summary>
    /// The mime type of drag data
    /// </summary>
    public string MimeType { get; set; }
    /// <summary>
    /// The drag data to send
    /// </summary>
    public string Data { get; set; }

    /// <summary>
    /// The mime types and drag data set
    /// </summary>
    public Dictionary<string, string> DataMap;
}
```
우와 같이 기존 DragInfo Structure를 새로 만들어서 구분하지 않고 기존 DragData에 DataMap Dictionary를 적용하여 코드의 응집도와 가독성 및 사용성을 높이는 작업을 하였습니다 😄

Reply:
깔끔하게 정리되어서 보기 좋은것 같습니다 😊
말씀하신대로 가독성과 사용성 측면에서 개선되어 보기 편하네요 👍